NewHorizons/Main.cs

```diff
@@ -350,9 +350,12 @@ private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
                     GravityCannonBuilder.InitPrefab();
                     ShuttleBuilder.InitPrefab();
 
-                    ProjectionBuilder.InitPrefabs();
```

Comment:
is there any other dlc only shit like that that shouldnt be called? idk

Reply:
ghost bird text but idc we'll wait for somebody to complain it broke